scripts/update-factory-manifest

```diff
@@ -191,4 +197,9 @@ git merge --no-edit -m "update-manifest: merge LmP ${latest}" ${latest} || abort
 echo ""
 echo "Automatic update successful!"
 
-git push origin HEAD:${remote_branch} && git push --tags origin
+if [ -z "${PREFIX}" ]; then
+    git push origin HEAD:${remote_branch} && git push --tags origin
+else
+    # only push tags matching the PREFIX
+    git push origin HEAD:${remote_branch} && git push origin refs/tags/${PREFIX}*:refs/tags/${PREFIX}*
```

Comment:
Yeah I thought about this.  While this isn't 100% necessary on the outbound if we're also scrubbing the inbound.  It did feel like the right thing to do since we didn't know what tags would be in the local repo at this stage.  I almost want to do this for the non-PREFIX path.. but that would have been a larger change and I need to think about it more.

Reply:
@ricardosalveti if it's too much, I can remove the push filter

Reply:
Fine to keep it.